languages/go/polar.go

```diff
@@ -135,7 +138,43 @@ func (p Polar) queryRule(name string, args ...interface{}) (*Query, error) {
 }
 
 func (p Polar) repl(files ...string) error {
-	return fmt.Errorf("Go REPL is not yet implemented")
+	reader := bufio.NewReader(os.Stdin)
+	for {
+		fmt.Print("query> ")
+		text, _ := reader.ReadString('\n')
+		text = util.QueryStrip(text)
+
+		if text == "" {
+			continue
+		}
+
+		ffiQuery, err := p.ffiPolar.NewQueryFromStr(text)
+		if err != nil {
+			fmt.Println(err)
+			continue
+		}
+		query := newQuery(*ffiQuery, p.host.Copy())
+		results, err := query.GetAllResults()
+		if err != nil {
+			fmt.Println(err)
+			continue
+		}
+		if len(results) == 0 {
+			fmt.Println(false)
+		} else {
+			for _, bindings := range results {
+				for k, v := range bindings {
+					switch v := v.(type) {
+					// print strings with quotes but not variables or other types represented by strings
+					case string:
+						fmt.Printf("%v = %#v\n", k, v)
```

Comment:
What does the `#` do?

Reply:
It prints the value as a go source literal, in this case that means adding quotes around it.